torch/optim/sgd.py

```diff
@@ -276,6 +276,7 @@ def _multi_tensor_sgd(params: List[Tensor],
                       maximize: bool,
                       has_sparse_grad: bool):
 
+    print(f"Shunting: call _multi_tensor_sgd {len(params)}") # TODO
```

Comment:
extra debug comment to be removed?

Reply:
oh i just noticed it’s wip, i see i see

Reply:
haha, yea, this is a PR for debugging purpose. It may be dropped later